test/evergreen.yml

```diff
@@ -40,6 +40,25 @@ functions:
       params:
         path: "WT_TEST-big-endian.tgz"
         destination: "wiredtiger/build_posix/test/format"
+  "fetch artifacts from many-coll" :
```

Comment:
I guess the reason for creating a new function (versus reusing the common `fetch artifacts`) is to avoid running into timeout error when extracting the tarball (assuming the size is huge)?

Reply:
This new function was created in order to retrieve a database (i.e `dbpath_100k_500.tgz`) that is available on our amazon s3 bucket.